LICENSE.md

```diff
@@ -0,0 +1,45 @@
+# LICENSE
+
+Licensing and Copyright Notice and Terms of Use
+
+Owner Notice: This software (including any associated website-service or
+downloadable source (code) was developed by the Lister Hill National Center
+for Biomedical Communications (LHNCBC), a research and development division
+of the U.S. National Library of Medicine (NLM), with the permission and based
+on the copyrighted content of the Regenstrief Institute.
+
+The following terms and conditions are based on the BSD open-source license.
+
+If publishing research that used this software, please include a citation that
+acknowledges it.
+
+Redistribution and use in source and binary forms, with or without modification,
+are permitted for commercial and non-commercial purposes and products alike,
+provided that the following conditions are met:
+
+  * Redistributions of source code shall retain the above Owner Notice, this
+  list of conditions and the following disclaimer, and display them prominently.
+  * Redistributions in binary form shall prominently reproduce the above Owner
+  Notice, this list of conditions and the following disclaimer in the
+  documentation and/or other materials provided with the distribution.
+  * Neither the names of the National Library of Medicine (NLM), the Lister
+  Hill National Center for Biomedical Communications (LHNCBC), the National
+  Institutes of Health (NIH), nor the names of any of the software contributors
+  may be used to endorse or promote products derived from this software without
+  specific prior written permission.
+
+THIS SOFTWARE IS PROVIDED BY THE OWNER AND CONTRIBUTORS "AS IS" AND ANY EXPRESS
+OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES OF
+MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT
+SHALL THE OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
+SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO,
+PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR
+BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER
+IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+POSSIBILITY OF SUCH DAMAGE.
+
+THE UCUM TABLE (IN ALL FORMATS), UCUM DEFINITIONS, AND SPECIFICATION ARE
```

Comment:
Since you are not using UCUM, you can leave this paragraph out.

Reply:
Fixed